SPECS/ruby/CVE-2025-6442.patch

```diff
@@ -0,0 +1,440 @@
+From 8b2fde21395a7978e3f469cbb6ea035e4061e06e Mon Sep 17 00:00:00 2001
```

Comment:
Please use proper open-source community etiquette here and restore original author information to the patch header.  If you made additional changes you can add your information as well.

From ee60354bcb84ec33b9245e1d1aa6e1f7e8132101 Mon Sep 17 00:00:00 2001
From: Jeremy Evans <code@jeremyevans.net>
Date: Tue, 25 Jun 2024 14:39:04 -0700
Subject: [PATCH] Require CRLF line endings in request line and headers

Disallow bare CR, LF, NUL in header and request lines. Tighten
parsing of request lines to only allow single spaces, as specified
in the RFCs.

Forcing this RFC-compliant behavior breaks a lot of tests, so
fix the tests to correctly use CRLF instead of LF for requests
(other than the specific checks for handling of bad requests).

Fixes #137
---

Reply:
Will do. I had been told previously to change the header when I made changes to a patch, so I have been operating under those instructions for a while. I will not change the headers from here on.